imagetest/README.md

```diff
@@ -72,6 +72,37 @@ Google Compute Engine startup script runner. The test files must specify a
 'cit' build constraint in order to prevent the presubmits on this repository
 from invoking the image tests.
 
+When writing tests to run against both Linux and Windows, it is preferred to
+use separate functions within the same test where appropriate based on
+differences between OSes (ex. powershell vs bash commands). This makes the
+test definitions easier to read and maintain.
+
+For example, if the test MyDefaultTest() needs to run different commands to
+achieve similar results:
+
+```go
+import runtime
+
+func TestConditionWindows() {
+      //Test something in Windows
+}
+func TestCondition() {
+      //Test something in Linux
+}
+
+func MyDefaultTest(
+      if runtime.GOOS == "windows" {
+		TestConditionWindows()
+	} else {
+		TestCondition()
+	}
+)
```

Comment:
Nit: `}` instead of `)`

Reply:
Oops thats definitely a syntax error, not a nit. Thanks for the catch.